userdocs/diagnostics/sending-closure-risks-data-race.md

```diff
@@ -0,0 +1,65 @@
+# Sending closure risks causing data races
+
+If a type does not conform to `Sendable` the compiler will enforce that each instance of that type is only accessed by one concurrency domain at a time. The compiler will also prevent you from capturing values in closures that are sent to another concurrency domain if the value can be accessed from the original concurrency domain too.
+
+For example:
+
+```swift
+class MyModel {
+  func perform() {
+    Task {
+      self.update()
+    }
+  }
+
+  func update() { ... }
+}
+```
+
+Building with complete concurrency checking will diagnose the capture of `self` in the task closure:
```

Comment:
The phrase "with complete concurrency checking" shows up multiple times through these notes - is it safe to assume that IF they're seeing these notes, they have such checking enabled? Some of the writing elsewhere in these notes seems to be assuming they're using complete concurrency checking.

If that's a safe assumption, this could be more direct with something like:
```The compiler diagnoses the capture of `self` in the task closure:```

Reply:
That's totally fair, and we can leave which configurations produce which errors to the migration guide. I'll remove the "with complete concurrency checking" language from all of the educational notes.